utils/scripts/ocm/README.md

```diff
@@ -0,0 +1,119 @@
+# Openshift Dedicated Cluster operations in AWS with OCM CLI and Openshift API
+Script to provision Openshift Dedicated (OSD) cluster on AWS and with other functionalities like creating identity providers(IDP) to access the cluster, installing/uninstalling add-ons and retrieving the cluster info with the aid of Openshift Cluster Manager CLI tool ([ocm-cli](https://github.com/openshift-online/ocm-cli.git))
+
+# Prerequisites
+- Python 3.x
+- [Install ocm cli](#Install%20OCM-CLI)
+- access_key and secret_access_key for the AWS account user *ocdCcsAdmin*
```

Comment:
could you change the above line to something like this:

  access_key and secret_access_key for the "osdCcsAdmin" AWS Identity and Access Management (IAM) user with the AdministratorAccess